Content.Server/Zombies/ZombifyOnDeathSystem.cs

```diff
@@ -92,6 +95,19 @@ public void ZombifyEntity(EntityUid target)
             //you're a real zombie now, son.
             var zombiecomp = AddComp<ZombieComponent>(target);
 
+            //Removes all other factions to prevent conflicting factions.
+            //This should ideally be coded to remove all factions rather than hardcoding them in... function does not exist.
+            //_faction.RemoveFaction(target);
+            _faction.RemoveFaction(target,"NanoTrasen", true);
+            _faction.RemoveFaction(target, "SimpleHostile", true);
+            _faction.RemoveFaction(target, "Syndicate", true);
+            _faction.RemoveFaction(target, "Xeno", true);
+            _faction.RemoveFaction(target, "Dragon", true);
```

Comment:
Just make a RemoveAllFactions method in faction system, it is a `Factions` hashset so it should be straightforward

Reply:
Dunno how.